picky/src/key/mod.rs

```diff
@@ -1018,20 +1093,6 @@ impl PublicKey {
     }
 }
 
-/// EC field's BigUint -> bytes conversion does not include leading zeros, therefore we need to
-/// expand the bytes to the curve's field size.
-fn expand_ec_field(bytes: Vec<u8>, curve: EcCurve) -> Vec<u8> {
```

Comment:
It's no longer needed.  `BoxedUnit::to_be_bytes_trimmed_vartime` gives the number of bytes, the size of the curve.